### PR TITLE
No longer wrap and run ct in Docker container

### PR DESCRIPTION
**Pull request by unguiculus**
With this major change, the action no longer wraps and runs ct in a Docker
container but simply installs it directly on the host. Default config
files that are part of the distro and Yamale and yamllint are installed as
well.

With this change, ct is directly exposed to the user with all options
directly available. It will be much easier now to integrate custom tools
using the new feature to run additional commands introduced in
helm/chart-testing/pull/283.

Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

### CODE REVIEW

File: ct.sh
Comment (davidkarlsen) [inline ct.sh:57]: This won't work so well with self-hosted runners.
How do you feel about using the APIs to cache tools and add them to the path, like done here: https://github.com/evryfs/cloudctl-action/blob/master/src/main.ts#L32

Comment (unguiculus) [reply ct.sh:57]: Interesting. I guess on a self-hosted runner you could just pre-install the tools and not use the action at all. Anyways, I'll look into it. Installing to the cache dir would probably be better. For the configs that won't work yet, but I think we could introduce and environment variable for that.

Comment (unguiculus) [reply ct.sh]: Updated. Depends on https://github.com/helm/chart-testing/pull/291

Comment (unguiculus) [reply ct.sh]: Tested locally and here: https://github.com/unguiculus/gh-pages-helm-chart-repo-example/runs/1336625053?check_suite_focus=true
The jobs has to fail until the mentioned PR is merged.

File: ct.sh
Comment (davidkarlsen) [inline ct.sh:58]: nit: Should maybe lookup architecture, although I guess close to all run on linux 64

File: ct.sh
Comment (davidkarlsen) [reply ct.sh]: I still prefer installing through actions (in contrast to putting it into the runner image/machine) as this can let the runners be quite slim, and whatever the workflow requires is handled through the workflow. This gives more control to the end-user, and different versions of same tool can run on same runner.

**Review by scottrigby — APPROVED** [review]
I saw the initial test for this and liked it. This PR looks right to me, though I haven't ever uses custom runners, so I'd defer to @davidkarlsen on that. Barring that, lgtm 💯

File: ct.sh
Comment (unguiculus) [reply ct.sh]: done

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**
